CMakeLists.txt

```diff
@@ -42,6 +42,8 @@ include_directories(include)
 include_directories(${MPI_DIR}/include)
 link_directories(${MPI_DIR}/lib)
 
+add_definitions(-DFIBER_ENABLE_HEFFTE)
```

Comment:
**This is a major issue**, if user don't have heFFTe installed and want to try, e.g., FFTE and AccFFT, this will not compile.
That is why the add_definitions in the current CMake are conditionally enable according to the cmake line that user give.

Reply:
That line was accidentally left in from when I was testing, and I've removed it.

I added back the add_definitions() since the documentation on automatic setting of those flags doesn't seem to be working.  I think it may be because it doesn't propagate the definitions to sub-directories.  We can refine it further , but it works for now.